### PR TITLE
fixed windows path support for error messages

### PR DESCRIPTION
**Pull request by AlexDM0**
Might as well do the pull request for #220

### CODE REVIEW

**Comment by ljharb** [comment]
Instead of detecting windows paths, can we just use the `path` module, which has OS-specific directory separators?

File: lib/test.js [hunk 1]
```diff
@@ -209,8 +209,14 @@ Test.prototype._assert = function assert (ok, opts) {
         var e = new Error('exception');
         var err = (e.stack || '').split('\n');
         var dir = path.dirname(__dirname) + '/';
```
Comment (ljharb) [inline lib/test.js]: specifically here - instead of using a hardcoded slash, we could use `path.sep`?

**Comment by AlexDM0** [comment]
Or we can write 4 lines of code and avoid yet another dependency.....

**Comment by ljharb** [comment]
a) more dependencies is always better than reinventing wheels.
b) path is built into node, and when browserified, is appropriately shimmed.

**Comment by AlexDM0** [comment]
Sure, path is also fine. As long as the bug is fixed I suppose.

**Comment by Raynos** [comment]
will merge if using portable `path` code.

I also dont have a way to verify or test this on windows itself.

**Comment by hgwood** [comment]
I have given another try to this fix: #295.

**Comment by ljharb** [comment]
@AlexDM0 Given #316, should this PR be closed, or rebased and re-evaluated?

**Comment by AlexDM0** [comment]
I think it can be closed, thanks!

**Pull request closed without merging**